source/includes/kubernetes/_k8_secrets.md

```diff
@@ -18,10 +18,10 @@ curl -X GET \
     {
       "id": "default-token-xxxmt/default",
       "apiVersion": "v1",
-      "data": {
-            "ca.crt": [],
-            "namespace": [],
-            "token": []
+      "encodedData": {
```

Comment:
How does this look like when the docs UI is built? Does it push away a lot of content. If so may adding something short as below might work.
```javascript
"ca.crt": 
    "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUN3akNDQWFxZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFTTVJBd0RnWURWUVFERXdkc......"
```

Reply:
2f34f7fcf8fff83c8201ee34f5d224ad3fb19c6e